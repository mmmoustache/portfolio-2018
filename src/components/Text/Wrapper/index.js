import styled from "styled-components";
import { convertRemToPixels as rem } from '../../../utils/utils';

export default styled.div`
  width: 100%;

  p {
    line-height: 1.6;
    margin: 0 0 ${rem('20px')};

    &:last-child {
      margin: 0;
    }
  }
`;
