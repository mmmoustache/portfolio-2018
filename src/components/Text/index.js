import React from 'react';
import PropTypes from 'prop-types';
import Wrapper from './Wrapper';

const Text = ({ children }) => <Wrapper>{children}</Wrapper>;

Text.propTypes = {
  children: PropTypes.node,
};

Text.defaultProps = {
  children: undefined,
};

export default Text;
