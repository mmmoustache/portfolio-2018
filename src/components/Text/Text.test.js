import React from 'react';
import renderer from 'react-test-renderer';
import Text from "../Text";

describe('Component: Text', () => {

  it('Should render successfully', () => {
    const component = renderer.create(
      <Text>Test</Text>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

});
