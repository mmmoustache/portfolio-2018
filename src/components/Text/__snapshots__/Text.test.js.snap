// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Text Should render successfully 1`] = `
<div
  className="sc-bdVaJa gbkfm"
>
  Test
</div>
`;
